src/components/body/bodyparts/AboutMe.css

```diff
@@ -1,23 +0,0 @@
-.aboutme{
-   margin: auto;
-   max-width: 500px;
-}
-
-.aboutme h2{
-    font-size: 32px;
-    margin: 0;
-    padding: 0;
-    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
-}
-p{
```

Comment:
realistically shouldnt be overridding the whole elements styling, make a class and assign it. You will get clashes